linux_os/guide/services/ssh/ssh_server/sshd_set_idle_timeout/rule.yml

```diff
@@ -9,6 +9,7 @@ description: |-
     To set an idle timeout interval, edit the following line in <tt>/etc/ssh/sshd_config</tt> as
     follows:
     <pre>ClientAliveInterval <b>{{{ xccdf_value("sshd_idle_timeout_value") }}}</b></pre>
+    while making sure that <tt>ClientAliveCountMax</tt> is set to <tt>0</tt>.
```

Comment:
ClientAliveCountMax needs to maintain itself in a new rule. One configuration change per rule only.